src/lib/components/layout/about.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+  import PrimaryButton from "../library/primary-button.svelte";
+
+  const items = [
+    "CDE adoption journeys",
+    "AI code assistants",
+    "Next-Gen Editors",
+    "Ephemeral databases",
+    "Generative AI",
+    "Secure remote access",
+    "Secrets Management",
+    "Developer Productivity",
+  ];
+</script>
+
+<div class="mt-xx-large md:mt-xxx-large max-w-7xl w-11/12 mx-auto space-y-10">
+  <h1 class="text-6xl font-normal text-headlines md:text-center">All about</h1>
+
+  <div class="flex flex-wrap gap-4 md:justify-center">
+    {#each items as item}
+      <p
+        class="border-blue border-2 m-0 rounded-xl px-4 py-2 font-medium text-white"
+      >
+        {item}
+      </p>
+    {/each}
+  </div>
+
+  <img src="/images/skateboard.webp" alt="CDE Universe Skateboard" />
+
+  <div class="items-start md:w-fit mx-auto">
+    <p class="text-xl font-bold text-white">Attend Conference</p>
```

Comment:
Could you reduce the spacing between the skateboard and the text underneath a bit? By that, it will look more like one section. 🙏

Reply:
I think space is itself in the image (top & bottom) empty space. I will crop the image and adjust accordingly.